tests/unit/test_models.py

```diff
@@ -0,0 +1,501 @@
+"""Unit tests for data models."""
```

Comment:
this seems to have significant overlap with the work @syedriko is doing on config.yaml here: https://github.com/openshift/lightspeed-service/pull/166

are you working talking with him?